### ISSUE TITLE
PhD

### ISSUE THREAD

**Issue by DrenfongWong** [zzm7000/zzm7000.github.io#1]
Hi Professor Zhao，I am quite interested in you research field. In fact，I work in the hypervisor，linux，crypto，compiler in intel now，Do you still open for recruiting？

**Comment by zzm7000** [comment]
@DrenfongWong Yes! Thank you for your interest! I have openings! Do you mind send me an email and schedule a quick chat?

**Comment by DrenfongWong** [comment]
> @DrenfongWong Yes! Thank you for your interest! I have openings! Do you mind send me an email and schedule a quick chat?

of course not. Thanks Professor Zhao, I have send you email :)
